### PR TITLE
Reconfigure access facet, push to the top, and make always open

### LINKED ISSUE #232: Rename "Online Access" Facet to "Access", put it at the top, and make it always open.
Technically, there is only one other way to access the materials and that's in person, which there is no toggle option for. It seems unnecessary for researchers to have to click the menu to click the option to see the refreshed results. If the drop down menu needs to be there, then should the rest of the materials be labeled with "In the Library" access like in the Catalog? (see screenshots of staging, current, and catalog examples).
staging site: Screen Shot 2020-04-07 at 10 47 42 AM
current site: Screen Shot 2020-04-07 at 10 52 38 AM
catalog example: Screen Shot 2020-04-07 at 10 54 32 AM

### PR DESCRIPTION
**Pull request by eliotjordan**
Closes #232 

Screen Shot 2020-11-19 at 5 31 23 PM

### CODE REVIEW

File: spec/features/facets_spec.rb [hunk 1]
```diff
@@ -20,4 +20,16 @@
       expect(page).to have_css("#facet-date_range_sim .missing", text: /Unknown/, visible: false)
     end
   end
+
```
Comment (sdellis) [inline spec/features/facets_spec.rb:23]: @eliotjordan does this test the new order? We probably want to check the text of `h3.facet-field-heading:first-child button` is "Access", right?

**Review by sdellis — CHANGES_REQUESTED** [review]

File: spec/features/facets_spec.rb (hunk 1 above)
Comment (eliotjordan) [reply spec/features/facets_spec.rb:23]: Yes. Good catch.

**Comment by eliotjordan** [comment]
@sdellis Updated.

**Review by sdellis — APPROVED** [review]
🎉

**Pull request merged**